README.md

```diff
@@ -13,6 +13,21 @@ The __filter__ package is generally provided as a folder with source (`.java`) a
 
 A sample set of images are provided in an __"images"__ directory.
 
+## Creating an executable jar file
+Project is already set up using the gradle build environment so we can just use that to create the jar file.
+1. Open up the CLI and change the directory to the projects root directory.
+2. Execute the command "gradle build"
```

Comment:
Why "gradle" and not "gradlew" (or "./gradlew")?
What is the name of the jar file and where is it to be found after it is built?
Does the "jar" task not work?  (If not, that would be a good new issue to open.)

Reply:
I wasn't able to get gradlew or jar to work for a while so they were excluded but I figured it out and have added them to the new change.